src/System.IO.FileSystem/tests/packages.config

```diff
@@ -1,7 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="System.Diagnostics.Debug" version="4.0.10-beta-22512" targetFramework="portable-net45+win+wpa81" />
   <package id="System.IO" version="4.0.10-beta-22512" targetFramework="portable-net45+win+wpa81" />
+  <package id="System.IO.FileSystem.Primitives" version="4.0.0-beta-22605" targetFramework="portable-net45+win" />
```

Comment:
Could you clarify please. You mean that project reference should be used instead of nuget package?